mrmustard/lab_dev/states/base.py

```diff
@@ -50,7 +51,7 @@
 )
 
 from ..circuit_components import CircuitComponent
-from ..circuit_components_utils import BtoPS
```

Comment:
Can you revert the changes to states?

Reply:
Yes, I did

Reply:
Weird theres still a diff? Did you do `git checkout develop ./mrmustard/lab_dev/states`?